### ISSUE TITLE
Pre-work Checklist: Developer: Kyle Angelich

### ISSUE THREAD

**Issue by KyleA99** [hackforla/website#5567]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (two per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by HackforLABot** [comment]
## Activity Log: KyleA99
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- KyleA99 assigned: https://github.com/hackforla/website/issues/5505 at 09/20/2023, 11:15 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/issues/5505#issuecomment-1728968396 at 09/21/2023, 12:00 AM PDT
- KyleA99 opened: https://github.com/hackforla/website/pull/5565 at 09/21/2023, 12:18 AM PDT
- KyleA99 opened: https://github.com/hackforla/website/issues/5567 at 09/21/2023, 12:30 AM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/5567 at 09/21/2023, 12:30 AM PDT
- KyleA99 closed as completed: https://github.com/hackforla/website/issues/5567#event-10431284884 at 09/21/2023, 12:30 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5563#issuecomment-1732471452 at 09/23/2023, 09:08 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5565#issuecomment-1732684836 at 09/24/2023, 03:30 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5563#pullrequestreview-1641131082 at 09/24/2023, 03:41 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5565#issuecomment-1732776891 at 09/24/2023, 06:42 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5565#issuecomment-1732909016 at 09/24/2023, 09:57 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5565#issuecomment-1734366168 at 09/25/2023, 12:49 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5563#issuecomment-1734391830 at 09/25/2023, 01:08 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5563#pullrequestreview-1647843313 at 09/27/2023, 08:49 PM PDT
- KyleA99 PR was merged: https://github.com/hackforla/website/pull/5565#event-10512470451 at 09/29/2023, 08:01 AM PDT
- KyleA99 issue closed by PR 5565: https://github.com/hackforla/website/issues/5505#event-10512470695 at 09/29/2023, 08:01 AM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/5597 at 09/29/2023, 09:29 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/issues/5597#issuecomment-1741674885 at 09/29/2023, 09:34 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/issues/5597#issuecomment-1747957428 at 10/04/2023, 07:51 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5665#issuecomment-1751584554 at 10/06/2023, 08:32 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5665#pullrequestreview-1663119735 at 10/07/2023, 04:06 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5692#issuecomment-1754703338 at 10/10/2023, 01:39 AM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5692#pullrequestreview-1668535607 at 10/10/2023, 11:13 AM PDT
- KyleA99 issue closed as completed: https://github.com/hackforla/website/issues/5597#event-10623333105 at 10/11/2023, 12:40 PM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/3529#issuecomment-1240951137 at 10/11/2023, 04:37 PM PDT
- KyleA99 unassigned: https://github.com/hackforla/website/issues/3529#issuecomment-1758696153 at 10/11/2023, 07:45 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/issues/3529#issuecomment-1758827760 at 10/11/2023, 07:47 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5692#issuecomment-1758858395 at 10/11/2023, 08:32 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5704#issuecomment-1762377177 at 10/13/2023, 04:36 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5692#issuecomment-1763234952 at 10/14/2023, 06:59 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5704#pullrequestreview-1678596087 at 10/14/2023, 07:57 PM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/5642 at 10/15/2023, 10:52 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/issues/5642#issuecomment-1769468243 at 10/18/2023, 03:09 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5733#issuecomment-1769517358 at 10/18/2023, 03:45 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5733#issuecomment-1770094958 at 10/18/2023, 10:47 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5733#pullrequestreview-1687440210 at 10/19/2023, 04:34 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/issues/5642#issuecomment-1773713669 at 10/21/2023, 01:11 AM PDT
- KyleA99 opened: https://github.com/hackforla/website/issues/5749 at 10/21/2023, 10:06 PM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/5749 at 10/21/2023, 10:06 PM PDT
- KyleA99 opened: https://github.com/hackforla/website/pull/5750 at 10/21/2023, 11:01 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5733#issuecomment-1774004984 at 10/21/2023, 11:06 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5750#issuecomment-1774043565 at 10/22/2023, 02:25 AM PDT
- KyleA99 closed: https://github.com/hackforla/website/pull/5750#event-10732926767 at 10/22/2023, 02:25 AM PDT
- KyleA99 closed as completed: https://github.com/hackforla/website/issues/5749#event-10732929922 at 10/22/2023, 02:28 AM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5733#pullrequestreview-1691445943 at 10/22/2023, 07:42 AM PDT
- KyleA99 issue closed as completed: https://github.com/hackforla/website/issues/5642#event-10733680405 at 10/22/2023, 10:20 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5768#issuecomment-1776314100 at 10/23/2023, 06:21 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5768#pullrequestreview-1694199269 at 10/24/2023, 12:38 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5789#issuecomment-1783645620 at 10/27/2023, 05:59 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5789#pullrequestreview-1702887488 at 10/28/2023, 07:14 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5733#pullrequestreview-1702990395 at 10/29/2023, 09:24 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5733#issuecomment-1784169333 at 10/29/2023, 10:06 AM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5733#pullrequestreview-1702996848 at 10/29/2023, 10:07 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5793#issuecomment-1784896000 at 10/30/2023, 03:28 AM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5793#pullrequestreview-1705413832 at 10/30/2023, 09:22 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5794#issuecomment-1788295880 at 10/31/2023, 07:19 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5794#issuecomment-1788392380 at 10/31/2023, 09:31 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5794#issuecomment-1789681545 at 11/01/2023, 02:02 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5794#pullrequestreview-1709084524 at 11/01/2023, 02:04 PM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/4296 at 11/02/2023, 12:56 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1790315670 at 11/02/2023, 01:56 AM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5794#issuecomment-1791522065 at 11/02/2023, 01:52 PM PDT
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5794#pullrequestreview-1711362189 at 11/02/2023, 03:43 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/pull/5828#issuecomment-1793652373 at 11/04/2023, 11:34 PM PDT
- KyleA99 commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1793741252 at 11/05/2023, 05:41 AM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5828#pullrequestreview-1713995157 at 11/05/2023, 06:21 AM PST
- KyleA99 commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1807047856 at 11/11/2023, 11:55 PM PST
- KyleA99 commented on: https://github.com/hackforla/website/pull/5903#issuecomment-1811627641 at 11/14/2023, 04:52 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5903#pullrequestreview-1731098225 at 11/14/2023, 05:47 PM PST
- KyleA99 commented on: https://github.com/hackforla/website/pull/5905#issuecomment-1811705368 at 11/14/2023, 06:29 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5905#pullrequestreview-1731163939 at 11/14/2023, 07:07 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5905#pullrequestreview-1731347315 at 11/14/2023, 10:57 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5904#pullrequestreview-1731593684 at 11/15/2023, 01:28 AM PST
- KyleA99 commented on: https://github.com/hackforla/website/pull/5907#issuecomment-1813528965 at 11/15/2023, 04:46 PM PST
- KyleA99 commented on: https://github.com/hackforla/website/pull/5908#issuecomment-1813617189 at 11/15/2023, 05:41 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5907#pullrequestreview-1733442111 at 11/15/2023, 07:34 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5908#pullrequestreview-1735855754 at 11/16/2023, 05:43 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5910#pullrequestreview-1738264782 at 11/17/2023, 10:19 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5912#pullrequestreview-1738274035 at 11/17/2023, 11:00 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5915#pullrequestreview-1738285352 at 11/17/2023, 11:43 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5908#pullrequestreview-1738530562 at 11/18/2023, 05:53 PM PST
- KyleA99 commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1817939103 at 11/19/2023, 10:28 AM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5904#pullrequestreview-1746943458 at 11/23/2023, 11:50 AM PST
- KyleA99 unassigned: https://github.com/hackforla/website/issues/4296 at 12/02/2023, 01:22 PM PST
- KyleA99 commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1837257078 at 12/02/2023, 01:23 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5950#pullrequestreview-1767300662 at 12/06/2023, 03:16 AM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/5950#pullrequestreview-1773564593 at 12/08/2023, 11:19 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/6075#pullrequestreview-1812364216 at 01/09/2024, 07:45 PM PST
- KyleA99 commented on: https://github.com/hackforla/website/pull/6148#issuecomment-1907278702 at 01/23/2024, 07:12 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/6148#pullrequestreview-1840741272 at 01/24/2024, 12:29 AM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/6148#pullrequestreview-1840744358 at 01/24/2024, 12:30 AM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/6148#pullrequestreview-1844951742 at 01/25/2024, 05:14 PM PST
- KyleA99 unassigned: https://github.com/hackforla/website/issues/5749 at 01/27/2024, 10:20 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/6148#pullrequestreview-1852386023 at 01/30/2024, 01:17 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/6148#pullrequestreview-1852475758 at 01/30/2024, 02:12 PM PST
- KyleA99 submitted review: https://github.com/hackforla/website/pull/6206#pullrequestreview-1860646336 at 02/02/2024, 04:01 PM PST

- KyleA99 opened: https://github.com/hackforla/website/issues/8719 at 08/04/2026, 08:02 PM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/8719 at 08/04/2026, 08:02 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/issues/8719#issuecomment-5200146114 at 08/05/2026, 08:45 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/issues/8719#issuecomment-5232719634 at 08/09/2026, 10:10 AM PDT
- KyleA99 commented: https://github.com/hackforla/website/issues/8719#issuecomment-5249290364 at 08/10/2026, 10:21 PM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/8656 at 08/10/2026, 10:36 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/issues/8656#issuecomment-5249462974 at 08/10/2026, 10:48 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/issues/8719#issuecomment-5249478316 at 08/10/2026, 10:51 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/issues/8656#issuecomment-5262637929 at 08/11/2026, 10:17 PM PDT
- KyleA99 opened: https://github.com/hackforla/website/pull/8736 at 08/13/2026, 06:02 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/issues/8719#issuecomment-5288275514 at 08/13/2026, 06:14 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/pull/8734#issuecomment-5310462386 at 08/16/2026, 05:20 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/pull/8734#issuecomment-5310718875 at 08/16/2026, 06:14 PM PDT
- KyleA99 issue was closed as completed: https://github.com/hackforla/website/issues/8656 at 08/18/2026, 06:43 PM PDT
- KyleA99 PR was merged: https://github.com/hackforla/website/pull/8736 at 08/18/2026, 06:43 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/pull/8735#issuecomment-5336887523 at 08/18/2026, 07:46 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/pull/8735#issuecomment-5336902751 at 08/18/2026, 07:49 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/pull/8735#issuecomment-5337045816 at 08/18/2026, 08:12 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/pull/8741#issuecomment-5337180193 at 08/18/2026, 08:32 PM PDT
- KyleA99 assigned: https://github.com/hackforla/website/issues/7763 at 08/18/2026, 08:34 PM PDT
- KyleA99 unassigned: https://github.com/hackforla/website/issues/7763 at 08/18/2026, 08:34 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/issues/8719#issuecomment-5351800648 at 08/19/2026, 10:33 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/pull/8744#issuecomment-5365972639 at 08/20/2026, 11:20 PM PDT
- KyleA99 commented: https://github.com/hackforla/website/pull/8745#issuecomment-5387250568 at 08/23/2026, 10:09 AM PDT
- KyleA99 unassigned: https://github.com/hackforla/website/issues/5505 at 08/25/2026, 03:57 AM PDT
- KyleA99 unassigned: https://github.com/hackforla/website/issues/5597 at 08/25/2026, 03:58 AM PDT
- KyleA99 unassigned: https://github.com/hackforla/website/issues/5642 at 08/25/2026, 03:58 AM PDT
